<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QWidget" name="groupBox" native="true">
     <property name="styleSheet">
      <string>
         QWidget {
           background-color: black; /* Background color for video player */
         }
        </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="controlLayout">
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string>
              QPushButton {
                background-color: #6200EA; /* Vibrant Purple */
                color: white;
                font-size: 14px;
                border-radius: 12px; 
                padding: 8px;
                border: none; 
              }
              QPushButton:hover {
                background-color: #3700B3; 
                transform: translateY(-2px); 
              }
              QPushButton:pressed {
                background-color: #230036; 
              }
             </string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string>
              QPushButton {
                background-color: #6200EA; 
                color: white;
                font-size: 14px;
                border-radius: 12px; 
                padding: 8px;
                border: none; 
              }
              QPushButton:hover {
                background-color: #3700B3; 
                transform: translateY(-2px); 
              }
              QPushButton:pressed {
                background-color: #230036; 
              }
             </string>
         </property>
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string>
              QPushButton {
                background-color: #6200EA; 
                color: white;
                font-size: 14px;
                border-radius: 12px; 
                padding: 8px;
                border: none; 
              }
              QPushButton:hover {
                background-color: #3700B3; 
                transform: translateY(-2px); 
              }
              QPushButton:pressed {
                background-color: #230036; 
              }
             </string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="timeControlLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_3">
         <property name="maximum">
          <number>2</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="volumeControlLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>100%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
